shell/platform/darwin/ios/framework/Source/FlutterTextInputPlugin.mm

```diff
@@ -1040,12 +1203,11 @@ - (UITextRange*)textRangeFromPosition:(UITextPosition*)fromPosition
 }
 
 - (NSUInteger)decrementOffsetPosition:(NSUInteger)position {
-  return fml::RangeForCharacterAtIndex(self.text, MAX(0, position - 1)).location;
+  return MAX(0, position - 1);
```

Comment:
Why was this removed?  This looks like it might be breaking UTF stuff?

Reply:
Well, it's actually needed for scribble to handle UTF stuff properly – for whatever reason, UIKit seems to want to inspect every position (even within a multi-byte character sequence) in order to function properly. I'll see about wrapping that in a check for `_scribbleInProgress` to make this more conservative

Reply:
Turns out that my _scribbleInProgress check wasn't working – I don't need this due to the short circuit in `- positionFromPosition`